src/jit_x64.c

```diff
@@ -0,0 +1,745 @@
+// Copyright (c) 2015 Big Switch Networks, Inc
```

Comment:
Drop the original licensing declaration since we are reworking the ubpf implementation. We can mention the following:
> This JIT implementation has undergone extensive modifications, heavily relying on the ubpf_jit_x86_64.[ch] from ubpf. The original ubpf_jit_x86_64.[ch] file served as the foundation and source of inspiration for adapting and tailoring it specifically for this JIT implementation. Therefore, credit and sincere thanks are extended to ubpf for their invaluable work.

Check src/map.[ch] for the way in which we provide credit for the previous work.